index.js

```diff
@@ -34,7 +34,7 @@ function Ora(options) {
 	this.stream = this.options.stream;
 	this.id = null;
 	this.frameIndex = 0;
-	this.enabled = (this.stream && this.stream.isTTY) && !process.env.CI;
+	this.enabled = ((this.stream && this.stream.isTTY) && !process.env.CI) || this.options.enabled;
```

Comment:
Should we maybe adapt the other tests where we're setting this manually to use this option too?